### DISCUSSION TITLE
Helm chart parameters reference isn't viewable on github

### DISCUSSION THREAD

**Discussion by calebwoofenden** [apache/airflow#26914] [General] [answered]
### What do you see as an issue?

This file https://github.com/apache/airflow/blob/main/docs/helm-chart/parameters-ref.rst has some Jinja code to auto-generate all the parameter references. This was changed from a plaintext list in this PR: https://github.com/apache/airflow/pull/15827. This seems like a good optimization, but I can't actually view the resulting text from the Jinja template on github. Is there a good way to view the generated file other than cloning the repo and passing it through Jinja on my local machine? I'd like to look at the parameters reference for this chart.

Tagging @jedcunningham author of the PR.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by jedcunningham** [comment, marked as answer]
Hey @calebwoofenden!

You can view it on the docs website here: https://airflow.apache.org/docs/helm-chart/stable/parameters-ref.html

Another (arguably better) interface is on ArtifactHUB: https://artifacthub.io/packages/helm/apache-airflow/airflow?modal=values-schema

**Reply by jedcunningham** [reply]
And if you mean for an unreleased chart, your best bet is to just look at the values.yaml directly.

**Reply by calebwoofenden** [reply]
Ah excellent, thanks!
